### PR TITLE
prepare for a 0.5.1 release

### PR DESCRIPTION
**Pull request by aiuto**
cc: @oquenchil @fweikert   This release includes a fix for the pipeline breakage on rules_cc.

### CODE REVIEW

**Review by nacl — APPROVED** [review]
LGTM.  Is a changelog update going to come in a follow-on change?

File: pkg/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
+# Release 0.5.1
+
+## New features
+- Add `pkg_deb(architecture_file)` to provide a way to set the Debian package architecture from the content of a file created at build time.  (#390)
+- Provide `pkg_install` for a "make install"-like experience in Bazel (#380)
+  - Note: This feature is still in development. Read the PR for more information.
+- Initial support for pkg_files* in pkg_zip (#373)
```
[the remark this answers is not in the document]
Comment (aiuto) [reply pkg/CHANGELOG.md:7]: Both done.

File: pkg/releasing/defs.bzl [hunk 2]
```diff
@@ -11,13 +11,15 @@ def print_rel_notes(
         mirror_host = None):
     tarball_name = ":%s-%s.tar.gz" % (repo, version)
     cmd = [
-        "$(location //releasing:print_rel_notes)",
+        # TODO(aiuto): Turn this into a rule, so we can use a label that is
+        # relative to this repo, not the enclusing repo.
+        "$(location @rules_pkg//releasing:print_rel_notes)",
```
[the remark this answers is not in the document]
Comment (aiuto) [reply pkg/releasing/defs.bzl]: I fixed it. The trick is to use Label("//releasing:print_rel_notes") in the macro. That resolve the label relative to the file the code appears in (@rules_pkg/releasing:defs.bzl).

**Comment by aiuto** [comment]
I missed the comment about "is a changelog update going to come ..."
Are you talking about something different than the updates to CHANGELOG.md?

**Comment by nacl** [comment]
> I missed the comment about "is a changelog update going to come ..."
> Are you talking about something different than the updates to CHANGELOG.md?

Nope, just the updates to CHANGELOG.md.  If you're going to do them later, that's okay with me.

**Pull request merged**